src/sentry/api/endpoints/project_events.py

```diff
@@ -63,3 +49,52 @@ def get(self, request, project):
             on_results=lambda x: serialize(x, request.user),
             paginator_cls=DateTimePaginator,
         )
+
+    def __search_events_snuba(self, request, project):
+        from functools32 import partial
+        from sentry.api.paginator import GenericOffsetPaginator
+        from sentry.api.serializers.models.event import SnubaEvent
+        from sentry.utils.snuba import raw_query
+
+        query = request.GET.get('query')
+        conditions = []
+        if query:
+            conditions.append(['message', 'LIKE', '%{}%'.format(query)])
+
+        data_fn = partial(
+            # extract 'data' from raw_query result
+            lambda *args, **kwargs: raw_query(*args, **kwargs)['data'],
+            start=timezone.now() - timedelta(days=90),
```

Comment:
This was carry over from #10025, I'm not sure if there's a reason this is exactly 90 days and not a variable retention period like the legacy implementation.